packages/block-editor/README.md

```diff
@@ -962,6 +964,26 @@ _Returns_
 
 -   `any`: Returns the value defined for the setting.
 
+### useSettings
+
+Hook that retrieves the given settings for the block instance in use.
+
+It looks up the settings first in the block instance hierarchy. If none are found, it'll look them up in the block editor settings.
+
+_Usage_
+
+```js
+const [ fixed, sticky ] = useSettings( 'position.fixed', 'position.sticky' );
```

Comment:
I'm a bit confused here because the PR description says

```js
const [ allowFixed, allowSticky ] = useSettings( [ 'position.fixed', 'position.sticky' ] );
```

Can both be used? It would be nice to be consistent so we don't need to normalise.

Reply:
Only the `useSettings( 'a', 'b' )` form can be used. The PR originally started with an array argument, `useSettings( settings: string[] )`, but then was changed to `useSettings( ...settings: string[] )` during review. That's why the PR description is out of date.

The main reason for that was that when reading a single setting, `useSetting( 'a' )` and `useSettings( 'a' )` are the same, except that for `useSettings` the return value is always an array (with one element).